arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/option/selective/OptionSelective.kt

```diff
@@ -1,155 +0,0 @@
-package arrow.core.extensions.option.selective
```

Comment:
All thse files `XXXSelective.kt` in `extensions` have to stay until the deprecation cycle has finished in `0.13.0`